common/lib/github_validators.py

```diff
@@ -7,6 +7,18 @@ class GithubValidationError(Exception):
     pass
 
 
+def validate_repo_and_commit(allowed_gitub_orgs, repo_url, commit, branch):
+    if not allowed_gitub_orgs:
```

Comment:
I realise this is preserving existing behaviour, but while we're here I'm just going to note that `allowed_github_orgs` as Falsey / empty list meaning "all github orgs are allowed" is the opposite of what I would have expected